README.md

```diff
@@ -2,7 +2,7 @@
 
 The aim of this project is to show how `fastify` can be used.
 
-**NB:** I don't pretend to teach you anything: this is my personal `fastify` usage!
+**NB:** this is my personal `fastify` usage, but if you want to contribute, do it!
```

Comment:
if it's personal, it should live under your own github repo :).

I will rephrase this that this is a WIP, and it does not reflect the best way we think fastify should be used, but it's rather an experiment to help driving fastify.

Reply:
I'm not good in writing docs. The worse thing I can do is this kind of phrase! so thanks 😃